packages/did-session/README.md

```diff
@@ -69,6 +69,17 @@ didsession.authorizations
 didsession.expiresInSecs
 ```
 
+You can also get an AccountId from a user's DID instead of the address and provider.
+
+```js
+import { DIDSession, getAccountIdByDID } from 'did-session'
+import { EthereumWebAuth } from '@didtools/pkh-ethereum'
+
+const ethProvider = // import/get your web3 eth provider
```

Comment:
Might be worth mentioning https://eips.ethereum.org/EIPS/eip-1193 somewhere in our docs to be extra clear.

Reply:
yeah ill add that to the did-session and eth-pkh docs, thanks!